src/set_theory/ordinal.lean

```diff
@@ -757,6 +757,23 @@ by simp only [le_antisymm_iff, ordinal.zero_le, and_true]
 protected theorem pos_iff_ne_zero {o : ordinal} : 0 < o ↔ o ≠ 0 :=
 by simp only [lt_iff_le_and_ne, ordinal.zero_le, true_and, ne.def, eq_comm]
 
+protected theorem not_lt_zero (o : ordinal) : ¬ o < 0 :=
+not_lt_of_le (ordinal.zero_le o)
```

Comment:
You can use `not_lt_zero'` from algebra/order/with_zero to do this, maybe that's easier in your applications?

Reply:
Can I? Neither ordinal addition nor multiplication are commutative. I can use `not_lt_bot` if I move this theorem down a bit, though.

Reply:
Oops, you're right